### PR TITLE
Add chownFilesTo

### PR DESCRIPTION
**Pull request by skjalgsm-statespace**
#### Changes

- Added chownFilesTo

#### Checklist

- [x] Read the contribution [guide](../CONTRIBUTING.md) and accept the [code](../CODE_OF_CONDUCT.md) of conduct
- [x] Readme (updated or not needed)
- [x] Tests (added, updated or not needed)

### CODE REVIEW

File: dist/steps/run_tests.sh [hunk 1]
```diff
@@ -114,3 +114,19 @@ for platform in ${TEST_PLATFORMS//;/ }; do
...
+if [[ -n "$CHOWN_FILES_TO" ]]; then
+  chown -R "$CHOWN_FILES_TO" "$UNITY_PROJECT_PATH"
+  chown -R "$CHOWN_FILES_TO" "$FULL_ARTIFACTS_PATH"
+  chown -R "$CHOWN_FILES_TO" "$FULL_COVERAGE_RESULTS_PATH"
+fi
+
+# Add read permissions for everyone to all artifacts
+chmod -R a+r "$UNITY_PROJECT_PATH"
+chmod -R a+r "$FULL_ARTIFACTS_PATH"
+chmod -R a+r "$FULL_COVERAGE_RESULTS_PATH"
```
Comment (webbertakken) [inline dist/steps/run_tests.sh:132]: Missing newline (recommend installing editorconfig plugin)

```suggestion
chmod -R a+r "$FULL_COVERAGE_RESULTS_PATH"

```

**Review by webbertakken — APPROVED** [review]

**Comment by webbertakken** [comment]
Can you confirm that this works?

You can use `game-ci/unity-test-runner@3cb1997ca99e78c8c206f67b4f45b0b5f334d5a4`

**Comment by skjalgsm-statespace** [comment]
Thanks, I tried that and it doesnt work. I get 

```
</test-run>chown: invalid user: ‘undefined’
chown: invalid user: ‘undefined’
chown: invalid user: ‘undefined’
Changing to "/github/workspace/_activate-license" directory.
/github/workspace/_activate-license /github/workspace
/github/workspace
```

File: dist/steps/run_tests.sh (hunk 1 above)
Comment (skjalgsm-statespace) [reply dist/steps/run_tests.sh]: Great, i installed the plugin and added the newline

**Comment by skjalgsm-statespace** [comment]
Would you mind having a look @webbertakken?

I am using this for my workflow:
```
      # Test
      - name: ✅ Run tests
        uses: statespacelabs/unity-test-runner@3cb1997ca99e78c8c206f67b4f45b0b5f334d5a4
        id: tests
        env:
          UNITY_LICENSE: ${{ secrets.UNITY_LICENSE }}
          UNITY_EMAIL: ${{ secrets.UNITY_EMAIL }}
          UNITY_PASSWORD: ${{ secrets.UNITY_PASSWORD }}
        with:
          chownFilesTo: '1001:1001'
          unityVersion: ${{ matrix.unityVersion }}
          testMode: ${{ matrix.testMode }}
          artifactsPath: ${{ matrix.testMode }}-artifacts
          githubToken: ${{ secrets.GITHUB_TOKEN }}
          checkName: ${{ matrix.testMode }} Test Results
          coverageOptions: 'generateAdditionalMetrics;generateHtmlReport;generateBadgeReport;assemblyFilters:+AimLab.*'
          gitPrivateToken: ${{ secrets.PAT }}
```

**Comment by skjalgsm-statespace** [comment]
I had forgot to save a file, which showed up in git as changed when I closed VS Code :) That fixed it

**Comment by webbertakken** [comment]
> That fixed it

Great! Can you confirm this works with the latest commit hash?

**Comment by skjalgsm-statespace** [comment]
> > That fixed it
> 
> Great! Can you confirm this works with the latest commit hash?

Yes, I used `statespacelabs/unity-test-runner@a629f6ebace0d7fcfd8ae94c081150210f94a172` to test with

**Pull request merged**